### DISCUSSION TITLE
Should I ignore some file types?

### DISCUSSION THREAD

**Discussion by DiggesT** [ai/size-limit#322] [General]
The [Verbum](https://github.com/ozanyurtsever/verbum) project is using size-limit GH actions [tests](https://github.com/ozanyurtsever/verbum/blob/master/.github/workflows/size.yml), and it's has an [error](https://github.com/ozanyurtsever/verbum/actions/runs/4913497566/jobs/8773756287#step:3:1030): "No loader is configured for ".ttf" files...". This files using by Katex library.

In [my](https://github.com/DiggesT/verbum-deploy-test/tree/fix/size) branch I [fix](https://github.com/DiggesT/verbum-deploy-test/actions/runs/4944446595/jobs/8839929344) it by [adding](https://github.com/DiggesT/verbum-deploy-test/commit/e95d107d6740af8fa8be022b633604d26c8243ee) ignore file types list: `["*.woff", "*.woff2", "*.ttf"]`.

Can you explain me please, why should I add ignore types list? And is that corrct to ignore some file types for analyse library size?

**Comment by ai** [comment]
You need to ignore it because webpack or esbuild can’t know how to build it by default.

We don’t ignore them out of the box, because main use case of using webpack from Size Limit is open source JS libraries (where you can’t use `ttf`). For webapp we recommend using `@size-limit/file` plugin which doesn’t have problem with `ttf`.

**Reply by DiggesT** [reply]
Thanks for your answer, it's clear for me.
